spec/models/quote_type_spec.rb

```diff
@@ -0,0 +1,15 @@
+require "rails_helper"
+
+RSpec.describe QuoteType, type: :model do
+  it { should validate_presence_of(:name) }
+  it { should validate_length_of(:name).is_at_most(255) }
+  describe "validations" do
+    subject { QuoteType.create(name: "Type 1") }
```

Comment:
Use faker gem to generate random names

Reply:
We are using at next models.
For this there was not installed Faker.

Reply:
The fact that this is not installed is not a reason to not use it here since you can add it here, also @Heirian's comment is not about future cases, but this case.